### DISCUSSION TITLE
How can I set up sub-spaces?

### DISCUSSION THREAD

**Discussion by mszulcz-mitre** [snapshot-labs/snapshot#3386] [Q&A]
I've read the documentation on [sub-spaces](https://docs.snapshot.org/spaces/sub-spaces) and tried following the instructions, but I can't seem to set one up.  Can more detailed instructions be provided?

Here's what I've tried:

- I created 2 spaces.  Call them MainSpace (ENS domain:  mainspace.eth) and SubSpace (ENS domain: subspace.eth). 
- From the home page of SubSpace, I clicked on 'Settings' and scrolled to 'Sub-spaces' section.
- In that section, I entered mainspace.eth in the field for 'Main space'.
- I clicked 'Save' on the right side of the window.
- I navigated to the space for MainSpace and expected to see tiles for MainSpace and SubSpace, as shown in the picture in the docs:
<img width="864" alt="image" src="https://user-images.githubusercontent.com/102264773/205823193-c0a6e8be-09dd-40fc-a7c4-7cc69297b991.png">

What am I doing wrong?

**Comment by samwhosung** [comment]
Enter the sub-space in the settings of the main space under sub-space, then enter the main space in the settings of the sub-space under main space.
